stack/services/src/main/java/org/apache/usergrid/services/notifications/TaskManager.java

```diff
@@ -73,7 +72,9 @@ public void completed(Notifier notifier, Receipt receipt, UUID deviceUUID, Strin
                 }
                 successes.incrementAndGet();
             }
+            em.incrementAggregateCounters( notification.getUuid(),null,notification.getType(),"completed",100000L );
```

Comment:
this increments the counter. by 10k or something. So change that to be sure that it isn't so high. Then do a test to make sure that increment properly increments.

Then find a way to attach these to the properties in notifications and get properly read back out. That seems mysterious to me .